src/MSBuild/Resources/Strings.resx

```diff
@@ -929,6 +929,120 @@
+                     Write out the value of one or more specified items and
+                     their associated metadata after evaluation without
+                     executing the build, or if either the -targets option
+                     or the -getTargetResult option is used, write out
+                     the values after the build.
+    </value>
+    <comment>
+      LOCALIZATION: "-getItem" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_45_GetTargetResultSwitch" Visibility="Public">
+    <value>  -getTargetResult:targetName,...
+                     Write out the output value of one or more targets and
+                     the specified targets will be executed.
+    </value>
+    <comment>
+      LOCALIZATION: "-getTargetResult" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_46_FeatureAvailabilitySwitch" Visibility="Public">
+    <value>  -featureavailability:featureName,...
+                     Check feature availability, The result is one of
+                     "Undefined", "Available", "NotAvailable" strings.
+                     (Short form: -fa)
+    </value>
+    <comment>
+      LOCALIZATION: "-featureavailability" and "-fa" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_47_TerminalLoggerSwitch" Visibility="Public">
+    <value>  -terminallogger
+                      Users are able to opt-in and use the TerminalLogger,
+                      EXCEPT when:
+                      - The terminal does not support ANSI codes or color
+                      - Output is redirected to a file or pipe
+                      (Short form: -tl)
+    </value>
+    <comment>
+      LOCALIZATION: "-terminallogger" and "-tl" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_51_GetResultOutputFileSwitch" Visibility="Public">
+    <value>  -getResultOutputFile:file
+                     Redirect output from get* into a file.
+
+                     Example:
+                     -getProperty:Bar -getResultOutputFile:Biz.txt
+                     This writes the value of property Bar into Biz.txt.
+    </value>
+    <comment>
+      LOCALIZATION: "-getResultOutputFile" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_48_TerminalLoggerParametersSwitch" Visibility="Public">
+    <value>  -termialLoggerParameters:default=[auto|false|true]
```

Comment:
It can wait but we will need to reorganize this to account for other parameters besides `default`

Reply:
Sorry. I only got the information from https://github.com/dotnet/msbuild/blob/main/documentation/terminallogger/Opt-In-Mechanism.md#terminallogger-parameters.  Not sure how to reorganize this one better to account for other parameters.

Reply:
@MichalPavlik  Could me help me how to reorganize this to account for other parameters besides default?

Reply:
@MichalPavlik Could me help me how to reorganize this to account for other parameters besides default?

Reply:
I think what I meant was that there will be (already are?) other terminal-logger parameters besides `default`, like verbosity and probably other things in the future. So this will need to be organized like `HelpMessage_13_ConsoleLoggerParametersSwitch` where both the parameter names and the values can be documented.

Reply:
Updated